packages/dds/tree/src/feature-libraries/chunked-forest/codec/codecs.ts

```diff
@@ -86,6 +163,7 @@ export function makeFieldBatchCodec(
 							context.schema.policy,
 							data,
 							context.idCompressor,
+							context.incrementalEncoderDecoder,
```

Comment:
We want to disable incremental encoding when selecting TreeCompressionStrategy.Compressed right?


```suggestion
							context.encodeType === TreeCompressionStrategyExtended.CompressedIncremental ? context.incrementalEncoderDecoder: undefined,
```

Reply:
The ForestSummarizer already does this so it's redundant to do it here. But since this doesn't just apply to summaries or a paritcular summarizer, it's fine to do it here too.